examples/scripts/SPI_MPI_psana.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#
+#SBATCH --job-name=SPI_MPI
+#SBATCH --output=spi_mpi.out
+#SBATCH --partition=anagpu
+#SBATCH --gres=gpu:1080ti:1
+#SBATCH --ntasks=8
+#SBATCH --tasks-per-node=4
+#SBATCH --mail-user=iris@slac.stanford.edu
+#SBATCH --mail-type=ALL
+
+
+CWD=/reg/neh/home/iris/Software/pysingfel/examples
+source ~iris/lcls2/setup_env.sh
```

Comment:
In the README, document how to set up lcls2 conda environment. 
Then you can remove this and insert a comment here saying you need psana2